test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Do you need this file?

Reply:
no, when rsepc didnt work as i wanted y tried that out, i can remove it now

Reply:
Same that's before.